### PR TITLE
Remove cncf/apisnoop prow interactions apart from #sig-arch + #sig-release supporting jobs

### PR DESCRIPTION
**Pull request by hh**
Closes https://github.com/cncf/apisnoop/issues/548

### CODE REVIEW

**Comment by hh** [comment]
@cblecker I want the jobs to stay, so I put back in the trigger plugin.

See https://github.com/kubernetes/test-infra/issues/12863#issuecomment-1139144446

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -1253,21 +1253,7 @@ plugins:
...
-    - dog # /bark replies with dog pictures
-    - goose
-    - heart
-    - help
-    - hold
-    - label
-    - lgtm
-    - retitle
-    - shrug
-    - size
```
Comment (hh) [inline config/prow/plugins.yaml:1270]: @BenTheElder removing all bot interactions other than trigger here ^^

**Comment by chases2** [comment]
Looks okay to me, but I'd like a second set of eyes
/approve
/assign @BenTheElder

**Comment by BenTheElder** [comment]
/lgtm
/approve

https://github.com/kubernetes/test-infra/issues/12863#issuecomment-1139150534 remains for discussion

**Pull request merged**